pkg/cassandra/util.go

```diff
@@ -32,7 +32,7 @@ func DatacenterStopping(dc *cassdcapi.CassandraDatacenter) bool {
 }
 
 func ComputeSystemReplication(kluster *api.K8ssandraCluster) SystemReplication {
-	rf := 3.0
+	rf := 5.0
```

Comment:
@jsanda in K8ssandra 1.x we have 5, in 2.x we had 3 so far. I updated to 5 FYI.

Reply:
Since that time I have heard multiple folks say RF = 3 should be sufficient for the system_auth which I made the change in k8ssandra-operator. Maybe it's worth revisiting.

Reply:
Thanks for the context! I'll set the default back to 3 in this PR, we can revisit that later imo.